### PR TITLE
Create plan to speed up HCB

### PR DESCRIPTION
**Pull request by maxwofford**
Why is this a markdown file? I like the review process for PRs & we don't have a central location for policy currently.

### CODE REVIEW

File: SLA.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Don't sweat it dog. 🕶️
+Let's break this boogie down to a 4-step together:
+
+1. As of our starting day, we declare Air~~brake~~Bankruptcy
+   - Over half of our issues haven't been seen in months, and may very well not be an issue anymore. dev time is more valuable than money
+   - Anything that is an issue, we'll hear about when it comes up again
+2. Regular triage check-in
+   - we can figure out if we want this to be async or sync. hq or online. shirts or skins.
+3. Tickets can be marked as "not an issue", or "one-time" issues, but we should be conciously deciding to skip them
```
Comment (albertchae) [inline SLA.md]: 👍

I think there will be certain classes of Airbrake errors we can change the code to stop reporting too, or maybe put at some threshold

File: SLA.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+But there's too many!!1! How will we do it??2?
+*The panic begins to set in....*
+
+Don't sweat it dog. 🕶️
+Let's break this boogie down to a 4-step together:
+
+1. As of our starting day, we declare Air~~brake~~Bankruptcy
+   - Over half of our issues haven't been seen in months, and may very well not be an issue anymore. dev time is more valuable than money
+   - Anything that is an issue, we'll hear about when it comes up again
```
Comment (albertchae) [inline SLA.md]: agree with this approach 💯

File: SLA.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Feel the beat? Ready to 🎵play along🎶?
+
+## 2. Error on lag
+
+Which brings me back to the original topic of this five-paragraph persuasive essay...
+
+The world is full of great tools for tracking this! New Relic! Lighthouse! Others!
+
+**And they're all giant complex tools that need to be learned.** The goal here is to speed up loadtimes, so let's skip that and hit the ground with something that we can actually start with today:
```
Comment (albertchae) [inline SLA.md]: Agree with this for now but I think we should still try to figure out how to use one of those tools to our advantage eventually

**Review by garyhtou — APPROVED** [review]
Sounds good

**Review by cjdenio — APPROVED** [review]
Looks great to me!

**Comment by cjdenio** [comment]
Closing as stale, though I'd love to revisit this in the future!

**Pull request closed without merging**